connector/saml/saml.go

```diff
@@ -315,16 +329,25 @@ func (p *provider) HandlePOST(s connector.Scopes, samlResponse, inResponseTo str
 	if assertion == nil {
 		return ident, fmt.Errorf("response did not contain an assertion")
 	}
+
+	// Even though subject is optional, without it we can pull the user ID,
```

Comment:
"Even though subject is optional, without it we can --> cannot pull the user ID,"